recipe-server/normandy/recipes/tests/test_api.py

```diff
@@ -441,36 +441,54 @@ def test_it_works(self, api_client):
     def test_approve(self, api_client):
         r = RecipeFactory()
         a = ApprovalRequestFactory(revision=r.latest_revision)
-        res = api_client.post('/api/v1/approval_request/{}/approve/'.format(a.id))
+        res = api_client.post('/api/v1/approval_request/{}/approve/'.format(a.id),
+                              {'comment': 'r+'})
         assert res.status_code == 200
 
         r.refresh_from_db()
         assert r.is_approved
```

Comment:
I suspect this is because the comment isn't on the revision object, but on the approval. Maybe something like this?

```python
assert r.latest_revision.approval.comment == 'r+'
```